config.schema.json

```diff
@@ -132,6 +133,11 @@
       "items": {
         "$ref": "#/$defs/cross_selling_source_token"
       }
+    },
+    "chain_fee_token_map": {
+      "type": "object",
+      "description": "Map of token denoms to their respective chain fee tokens",
```

Comment:
Just use `Map of blockchain to onchain fee token(carbon denom equivalent) and actual decimals`